test/backend/test_onnx_backend.py

```diff
@@ -30,11 +30,17 @@ def skip_not_implemented_ops_test(test):
   onnxtf_ops_list = get_onnxtf_supported_ops()
   onnx_ops_list = get_onnx_supported_ops()
   for op in onnx_ops_list:
+    op_name = op
+    for i in range(1, len(op)):
+      if op[i].isupper():
+        op_name = op_name.replace(op[i], '_' + op[i])
```

Comment:
Does it cover the same upper case character appearing twice in the op name, such as RecurrentNeuralNetwork?